commit f5a457f684fe3ef41354975864757260270fa22a (refs/changes/36/1736/6)
Author: Kody A Kantor <kody@kkantor.com>
Date:   2018-05-08T20:03:48+00:00 (1 year, 5 months ago)
    
    MANTA-2980 electric-moray should report moray shard for operations like 'putObject'
    Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>
    Approved by: Jordan Hendricks <jordan.hendricks@joyent.com>

diff --git a/lib/moray.js b/lib/moray.js
index ca89840..99d8f4f 100644
--- a/lib/moray.js
+++ b/lib/moray.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 //
@@ -436,7 +436,7 @@ function put(options, cb) {
         headers: opts.headers
     }, 'Moray.' + op + ': entered');
 
-    client.putObject(bucket, key, md, opts, function (err) {
+    client.putObject(bucket, key, md, opts, function (err, data) {
         if (err) {
             log.debug({
                 err: err,
@@ -457,7 +457,7 @@ function put(options, cb) {
                 key: key,
                 requestId: opts.requestId
             }, 'Moray.' + op + ': done');
-            cb(null, md);
+            cb(null, md, data);
         }
     });
 }
